test/recipes/95-test_external_keyfiles.t

```diff
@@ -0,0 +1,61 @@
+#! /usr/bin/env perl
+# Copyright 2015-2021 The OpenSSL Project Authors. All Rights Reserved.
+#
+# Licensed under the Apache License 2.0 (the "License").  You may not use
+# this file except in compliance with the License.  You can obtain a copy
+# in the file LICENSE in the source distribution or at
+# https://www.openssl.org/source/license.html
+
+use File::Copy;
+use OpenSSL::Test;
+use OpenSSL::Test::Utils;
+use OpenSSL::Test qw/:DEFAULT data_file bldtop_dir bldtop_file srctop_dir srctop_file cmdstr/;
+
+setup("test_external_keyfiles");
+
+plan skip_all => "No external tests in this configuration"
+    if disabled("external-tests");
+
+my @p12_files = glob(srctop_file("keyfile-corpus", "*.p12"));
+
+# Skip malformed files
+@p12_files = grep(!/malformed/, @p12_files);
+
+# Skip files with separate cipher/mac passwords
+@p12_files = grep(!/pass-cipher/, @p12_files);
+
+# Skip files with an empty password - pkcs12 app cannot handle this
+@p12_files = grep(!/pass\(empty/, @p12_files);
```

Comment:
it can handle them, use `password-newline.txt` created like so: `echo > password-newline.txt` as password file